aggregations/db_tables/daily_add_key_by_app_count.py

```diff
@@ -0,0 +1,94 @@
+import datetime
+
+from . import DAY_LEN_SECONDS, daily_start_of_range, time_json
+from ..periodic_aggregations import PeriodicAggregations
+
+
+class DailyAddKeyByAppCount(PeriodicAggregations):
+    def dependencies(self) -> list:
+        return ["near_ecosystem_entities"]
+
+    @property
+    def sql_create_table(self):
+        return """
+            CREATE TABLE IF NOT EXISTS daily_add_key_by_app_count
+            (
+                collected_for_day  DATE NOT NULL,
+                app_id TEXT NOT NULL,
+                receipt_receiver_account_id TEXT NOT NULL,
+                add_key_count INTEGER NOT NULL,
+                PRIMARY KEY (collected_for_day, app_id, receipt_receiver_account_id )
+            );
+            CREATE INDEX IF NOT EXISTS daily_add_key_by_app_count_idx
+                ON daily_add_key_by_app_count (collected_for_day, add_key_count DESC)
+        """
+
+    @property
+    def sql_drop_table(self):
+        return """
+            DROP TABLE IF EXISTS daily_add_key_by_app_count
+        """
+
+    @property
+    def sql_select(self):
+        all_apps = """
+            SELECT token as contract_id, 
+            slug as app_id
+            FROM public.near_ecosystem_entities e, unnest(string_to_array(e.contract, ', ')) s(token)
+            WHERE app = TRUE AND length(contract)>0
```

Comment:
It is super hard to read. This construction
```
unnest(string_to_array(e.contract, ', ')) s(token)
```
could not appear if you design `near_ecosystem_entities` properly.

`near_ecosystem_entities.contract` has `text` type and it is named in a single form, why do we try to split it by comma?

Reply:
It's the first time when I see `s(token)` construction. Can you please explain what does it do? What's the difference between your usual way to create the alias (`as token`) and this way to create the alias?

Reply:
`SELECT token as contract_id` confuses me a lot.
You created `unnest(string_to_array(e.contract, ', ')) s(token)` with `token` name, then you renamed it in the same expression by `token as contract_id`, why do you use so complicated logic?

I also have some domain-specific questions. What is the token and what is the contract id? Why do we say that token and contract_id are the same things?

Reply:
You and Frol are correct, I can remove token keyword.  That is only necessary in the MSSQL dialect.  I just updated.

I agree, near_ecosystem_entities could have a better layout.  That is the data being provided today.  
It is the data provided from the near/ecosystems repository without alteration.  For now this is the best data that is available.  In general, I attempt to create an extract-load-transform where the data gets stored before I do transformations.  

These naming conventions like "slug", "contract", ect...  come from that near/ecosystems repository.  Please feel free to suggest any name changes, this will be your baby long-term.